### DISCUSSION TITLE
关于跟随距离和饰品附魔的建议

### DISCUSSION THREAD

**Discussion by Jedidiah-Zhang** [TartaricAcid/TouhouLittleMaid#343] [Suggestions & Ideas]
_1.16.5 版本_
1. 跟随模式下的女仆貌似跟随玩家的优先级高于工作和捡垃圾。我发现玩家一旦移动，女仆就会放弃手头上的事，尝试走到玩家身边。这就导致玩家移动中女仆几乎没有工作效率。这在某些模式下不是很方便。另外跟随距离目测大概是7-8格，但是女仆会反复尝试走到这个范围外去工作和捡垃圾，造成的结果就是女仆在目标和玩家之间反复横跳。

7-8格的活动范围我觉得有点小了，可以适当扩大。另寻路模式下女仆总喜欢贴脸，有时会影响操作。希望寻路下的女仆平时可以在范围内自由游走或工作，一旦超出范围寻路至玩家范围内为止，而非寻路到玩家。

2. 饰品目前可以附魔耐久和经验修补。但是实际测试下来好像在女仆身上时不会触发，希望可以。（我没正经做实验，只是用下来的感觉，如果本身可以出发，当我没说）

**Comment by TartaricAcid** [comment]
建议性质的内容不要发 Issues 了，发到 Discussions
